util/bloom/bloom_test.go

```diff
@@ -169,18 +170,21 @@ func TestMarshalJSON(t *testing.T) {
 	}
 }
 
-func BenchmarkCreateLargeFilter(b *testing.B) {
+const largeFilterElements = 150000
+
+// BenchmarkCreateLargeBloomFilter should have the same structure as xor_test.go BenchmarkCreateLargeXorFilter
+func BenchmarkCreateLargeBloomFilter(b *testing.B) {
```

Comment:
you "broke" the barrier here between the bloom filter, and the xor filter by including the Marshaling as part of the "PutXX" benchmarks. I think that for your new testing it is correct, however, why don't you retain the old one as a separate benchmark ?

Reply:
It changes the test but maintains comparability between xor and bloom. The xor filter does the majority of its work in MarshalBinary() so I had to put that in its benchmark, but then it seemed only fair to compare to to the bloom filter going through the same steps. Do we keep archived history of performance on benchmarks? In this case the benchmark isn't being used to see if changing code makes it more optimized, but to compare two different implementations.

Reply:
exactly. I agree that your changes are good and needed in order to compare "apples" to "apples".
I don't have any particular sentiment for the existing benchmark, just figured there is no good reason to "drop it" unless it's in our way. Just call the old one "BenchmarkSetOnLargeBloomFilter".

Reply:
I tried adding back an unaltered benchmark but it seems out of place. It doesn't add anything other than being a copy of the new Benchmark but without MarshalBinary()

Reply:
Since we don't care about the historic performance of the benchmark from before this PR to after it I think it's okay to change the benchmark. I don't think we gain anything by having a duplicate of the benchmark with one line different.